### PR TITLE
Microsoft R Open (MRO) Mac OS X Package

### PR DESCRIPTION
**Pull request by marcionicolau**
Add MRO to Homebrew/Casks from official repo https://mran.revolutionanalytics.com/download
This last update MRO-3.2.3 add support to El Capitan

### CODE REVIEW

**Comment by adidalal** [comment]
Replaces https://github.com/caskroom/homebrew-cask/blob/master/Casks/rro.rb?

File: Casks/mro.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'mro' do
+  version '3.2.3'
+  sha256 'c15ba6cd1f9fb910ef964e571c890c4a5db4f3d4c7850db5008802226c834ef0'
+
+  url "https://mran.revolutionanalytics.com/install/mro/3.2.3/MRO-#{version}-OSX.pkg"
+  name 'MRO'
+  homepage 'https://mran.revolutionanalytics.com'
+  license :gpl # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  app 'Microsoft R Open.app'
```
Comment (adidalal) [inline Casks/mro.rb]: A `.pkg` is installed, you can't use the `app` stanza.

File: Casks/mro.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'mro' do
+  version '3.2.3'
+  sha256 'c15ba6cd1f9fb910ef964e571c890c4a5db4f3d4c7850db5008802226c834ef0'
+
+  url "https://mran.revolutionanalytics.com/install/mro/3.2.3/MRO-#{version}-OSX.pkg"
```
Comment (adidalal) [inline Casks/mro.rb]: You can use "version" more than once

File: Casks/mro.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'MRO'
+  homepage 'https://mran.revolutionanalytics.com'
+  license :gpl # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (adidalal) [inline Casks/mro.rb]: Remove the comment, as lic. filled in

File: Casks/mro.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+cask 'mro' do
+  version '3.2.3'
+  sha256 'c15ba6cd1f9fb910ef964e571c890c4a5db4f3d4c7850db5008802226c834ef0'
+
+  url "https://mran.revolutionanalytics.com/install/mro/3.2.3/MRO-#{version}-OSX.pkg"
+  name 'MRO'
```
Comment (adidalal) [inline Casks/mro.rb]: Use the full name here

**Comment by adidalal** [comment]
Please make the changes as requested, though feel free to use rro.rb as a model and ask for help if needed.

**Comment by marcionicolau** [comment]
Hi @adidalal I'll make all changes. About replacement, yes. The MRO substitute RRO one.

**Comment by adidalal** [comment]
pkgutil names have changed, use:

```
com.microsoft.pkg.untitled_package
com.microsoft.rservices.mro2
com.microsoft.rservices.mro1
```

Also, I don't use this app, so if you could verify those directories, that would be great.

**Comment by marcionicolau** [comment]
@adidalal I completed all changes and verify directories.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in c129758c8086d86b3d33cc8c2045b4a26bee2a83. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**